### DISCUSSION TITLE
How to avoid quotes in Sql.Expression

### DISCUSSION THREAD

**Discussion by lucas-garrido** [linq2db/linq2db#3403] [Q&A] [answered]
Hi ! 
Simple example :
```c#
    public static partial class SqlExpr
    {
        [Sql.Expression("CONTAINS(({0}), {1}, FUZZY(0.8,'similarCalculationMode=substringsearch'))", ServerSideOnly = true, IsPredicate = true, CanBeNull = false)]
        public static bool TextSearch(string columns, string value) => throw new InvalidOperationException();

    }
```

this generate  `AND CONTAINS(('KUNNR, NAME1'), :"searchText", FUZZY(0.8,'similarCalculationMode=substringsearch'))`
which trigger error `The first argument should be a column`

how can i avoid these quotes ?
🙏

**Comment by MaceWindu** [comment, marked as answer]
You need to use Sql.Extension, check  https://github.com/linq2db/linq2db/blob/master/Source/LinqToDB/DataProvider/SqlServer/SqlServerExtensions.cs#L676 for example of similar FTS declaration for sql server. Not sure which db you use, as I don't see 3-parameter version of CONTAINS there with non-language parameter

**Reply by MaceWindu** [reply]
also test with call example : https://github.com/linq2db/linq2db/blob/master/Tests/Linq/Linq/FullTextTests.SqlServer.cs#L1902-L1916
